ssclj/jar/src/com/sixsq/slipstream/ssclj/migrate/user_cred.clj

```diff
@@ -0,0 +1,190 @@
+;;
+;; To run this script:
+;; java -cp .../ssclj-XXX-SNAPSHOT-standalone.jar com.sixsq.slipstream.ssclj.migrate.user-cred
+;;
+(ns com.sixsq.slipstream.ssclj.migrate.user-cred
+  (:require
+    [korma.core :as kc]
+    [korma.db :as kdb]
+    [clojure.java.io :as io]
+    [environ.core :as environ]
+    [com.sixsq.slipstream.ssclj.app.routes :as routes]
+    [sixsq.slipstream.client.api.cimi :as cimi]
+    [sixsq.slipstream.client.sync :as sync]
+    [com.sixsq.slipstream.ssclj.resources.common.utils :as u]
+    [sixsq.slipstream.client.api.authn :as authn]
+    [clojure.edn :as edn])
+  (:gen-class))
+
+(def ^:const keys-cred-full [:href :key :secret :connector :domain-name :tenant-name])
+(def ^:const keys-cred-nuvlabox (vec (remove #{:domain-name :tenant-name} keys-cred-full)))
+(def ^:const keys-cred-stratuslab (vec (remove #{:domain-name :tenant-name} keys-cred-full)))
+(def ^:const keys-cred-otc keys-cred-full)
+(def ^:const keys-cred-openstack keys-cred-full)
+(def ^:const keys-cred-opennebula (vec (remove #{:domain-name :tenant-name} keys-cred-full)))
+(def ^:const keys-cred-exoscale (vec (remove #{:tenant-name} keys-cred-full)))
+(def ^:const keys-cred-ec2 (vec (remove #{:domain-name :tenant-name} keys-cred-full)))
+
+
+(def ^:const cat-nuvlabox #{"nuvlabox-arthur-harden", "nuvlabox-bertil-ohlin",
```

Comment:
We should move this hardcoded maps specific to nuvla in separated config edn file(s) and load them from there. It's too much hardcoded like that. We don't have time to do this for this release, so I will merge it like that but worth to change this next week